docs/reference/query-rules/apis/list-query-rulesets.asciidoc

```diff
@@ -61,16 +61,33 @@ A sample response:
     "results": [
         {
             "ruleset_id": "ruleset-1",
-            "rules_count": 10
+            "rule_total_count": 10,
```

Comment:
Do you think it's good to have those details sent back to the user as part of the list query rules endpoint?

Maybe we could provide a parameter for the operation with "details=true" for this, or create a separate operation for obtaining telemetry.

That would keep the API response cleaner and separate from our telemetry needs.

WDYT?

Reply:
I'm OK with returning them in the list response since it's still a summary. If you feel strongly that it should be parameterized let's chat!

Reply:
I think we're changing the API because of telemetry. We didn't see the need for this before, and that got me thinking.

If you feel this is good for the users to have, so be it!

Reply:
TBH I think it's useful information, when the list call was created we didn't have the additional criteria types.